packages/CalloutParagraph/CalloutParagraph.jsx

```diff
@@ -6,6 +6,7 @@ import { safeRest } from '@tds/util-helpers'
 
 import Paragraph from '@tds/core-paragraph'
 import { colorTelusPurple } from '@tds/core-colours'
+import * as DecorativeIcons from '@tds/core-decorative-icon'
```

Comment:
Can we pass the icon element directly via the `icon` prop?

See Link icons - https://tds.telus.com/components/index.html#!/Links/Link/5